code/game/machinery/doors/ministile.dm

```diff
@@ -31,7 +31,7 @@
 	. = ..()
 	if(istype(mover) && (mover.pass_flags & PASSGLASS))
 		return TRUE
-	if(istype(mover, /mob/living/simple_animal/bot))
+	if(istype(mover, /mob/living/simple_animal/bot) || istype(mover, /obj/vehicle/ridden/wheelchair))
```

Comment:
Will this not make turnstiles AA for anyone in a wheelchair?

Reply:
i for one think this is an excellent change

Reply:
This does.